package.json

```diff
@@ -86,71 +85,71 @@
-    "@babel/preset-react": "7.17.12",
-    "@babel/preset-typescript": "7.18.6",
+    "@babel/preset-env": "7.21.5",
+    "@babel/preset-react": "7.18.6",
+    "@babel/preset-typescript": "7.21.5",
+    "@types/classnames": "2.3.1",
     "@types/lodash": "4.14.192",
     "@types/react-router-dom": "5.3.3",
     "@types/react-text-truncate": "0.14.1",
     "@types/react-window": "1.8.5",
-    "@typescript-eslint/eslint-plugin": "5.48.1",
-    "@typescript-eslint/parser": "5.48.0",
-    "autoprefixer": "10.4.7",
-    "babel-eslint": "10.1.0",
-    "babel-loader": "8.2.5",
+    "@types/webpack-livereload-plugin": "^2.3.3",
+    "@typescript-eslint/eslint-plugin": "5.59.1",
+    "@typescript-eslint/parser": "5.59.1",
+    "autoprefixer": "10.4.14",
+    "babel-loader": "9.1.2",
     "babel-plugin-inline-classnames": "2.0.1",
     "babel-plugin-transform-react-remove-prop-types": "0.4.24",
     "core-js": "3.30.1",
-    "css-loader": "6.7.1",
+    "css-loader": "6.7.3",
     "css-modules-typescript-loader": "4.0.1",
-    "esformatter": "0.11.3",
-    "eslint": "8.15.0",
-    "eslint-config-prettier": "8.6.0",
+    "eslint": "8.39.0",
+    "eslint-config-prettier": "8.8.0",
     "eslint-plugin-filenames": "1.3.2",
-    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-import": "2.27.5",
     "eslint-plugin-prettier": "4.2.1",
-    "eslint-plugin-react": "7.30.0",
+    "eslint-plugin-react": "7.32.2",
     "eslint-plugin-react-hooks": "4.6.0",
-    "eslint-plugin-simple-import-sort": "8.0.0",
+    "eslint-plugin-simple-import-sort": "10.0.0",
     "file-loader": "6.2.0",
-    "filemanager-webpack-plugin": "7.0.0-beta.0",
-    "fork-ts-checker-webpack-plugin": "7.2.14",
-    "html-webpack-plugin": "5.5.0",
+    "filemanager-webpack-plugin": "8.0.0",
+    "fork-ts-checker-webpack-plugin": "8.0.0",
+    "html-webpack-plugin": "5.5.1",
     "loader-utils": "^3.2.1",
-    "mini-css-extract-plugin": "2.6.0",
-    "postcss": "8.4.14",
+    "mini-css-extract-plugin": "2.7.5",
+    "postcss": "8.4.23",
     "postcss-color-function": "4.1.0",
-    "postcss-loader": "7.0.0",
-    "postcss-mixins": "9.0.2",
-    "postcss-nested": "5.0.6",
-    "postcss-simple-vars": "6.0.3",
+    "postcss-loader": "7.3.0",
+    "postcss-mixins": "9.0.4",
+    "postcss-nested": "6.0.1",
+    "postcss-simple-vars": "7.0.1",
     "postcss-url": "10.1.3",
-    "prettier": "2.8.2",
+    "prettier": "2.8.8",
     "require-nocache": "1.0.0",
-    "rimraf": "3.0.2",
+    "rimraf": "5.0.0",
     "style-loader": "3.3.1",
-    "stylelint": "14.8.2",
-    "stylelint-order": "5.0.0",
+    "stylelint": "15.6.0",
+    "stylelint-order": "6.0.3",
+    "terser-webpack-plugin": "5.3.7",
     "ts-loader": "9.4.2",
-    "typescript-plugin-css-modules": "4.1.1",
+    "typescript-plugin-css-modules": "5.0.1",
     "url-loader": "4.1.1",
     "webpack": "5.81.0",
     "webpack-cli": "5.0.2",
     "webpack-livereload-plugin": "3.0.2",
     "worker-loader": "3.0.8"
   },
+  "resolutions": {
```

Comment:
Why is this now necessary? Also seems like yet another thing that needs to be updated.

Reply:
It's temporary, can be removed as soon as stylelint is done with their ESM transition.